src/proxy.rs

```diff
@@ -10,15 +10,6 @@ use tokio::net::TcpStream;
 
 use crate::net;
 
-/// Proxy the inbound stream to a target address.
-pub async fn proxy(
```

Comment:
Where did this go?

This is important for proxying. and (I believe) the `forward` occupy method.

Reply:
`forward` occupy method (and all other proxy usages I could find) use `proxy_with_queue`. I was getting warnings that proxy was unused after removing the only usage of it I could find so I removed it.

It was used before parsing any login packets if the server has started to directly proxy the connection immediately after getting it. But now that we need to parse the login packets we do build the received packet queue.

I considered either leaving the method there unused or commenting it out, what would you recommend.

Reply:
>It was used before parsing any login packets if the server has started to directly proxy the connection immediately after getting it. But now that we need to parse the login packets we do build the received packet queue.

Ah, thanks! You're right. We can't directly forward anymore because we need to read and route the handshake packet. Makes sense!

>I considered either leaving the method there unused or commenting it out, what would you recommend.

Then just remove it, like you did! Then there's no reason to leave it in.